ksql-engine/src/test/java/io/confluent/ksql/util/ExpressionTypeManagerTest.java

```diff
@@ -83,6 +83,18 @@ public void testComparisonExpr() {
         Assert.assertTrue(exprType2.type() == Schema.Type.BOOLEAN);
     }
 
+    @Test
+    public void testLikeExpr() {
```

Comment:
nit: we try to use a "should<behaviour under test>" naming convention for our tests. As you can see, not all the tests have been updated accordingly :). But we should use it for new tests. So, for this test we could name it something like:

```shouldEvaluateBooleanSchemaForLikeExpression```